pandora-client-web/.env.template

```diff
@@ -1,6 +1,5 @@
 # Server connection config
-DIRECTORY_ADDRESS="http://127.0.0.1:25560"
-#DIRECTORY_ADDRESS="https://directory.project-pandora.com"
```

Comment:
is there any reason not leave this here commented out?

Reply:
The reason is that as we made the production server policies more strict, it will no longer work - server will reject the connection from non-production client.